Tests/RxCodableTests/RxCodableTests.swift

```diff
@@ -76,4 +76,15 @@ class RxCodableTests: XCTestCase {
     XCTAssertEqual(try Single.just(jsonString).map([User].self).toBlocking().first()!, users)
     XCTAssertEqual(try Maybe.just(jsonString).map([User].self).toBlocking().first()!, users)
   }
+  
+  func testMapCodableFromDictionary() {
+    let dictionary:[String:Any] = [
+      "id":123,
+      "name":"iamchiwon",
```

Comment:
Can you please add a whitespace after the colon(:)?

Reply:
ok. 
it'll be like below

"id": 123,
"name": "iamchiwon",